update/genesis/import_test.go

```diff
@@ -131,7 +128,12 @@ func TestNewStateImport(t *testing.T) {
 func TestImportAll(t *testing.T) {
 	t.Parallel()
 
-	folderName := "files"
+	folderName := "./files"
```

Comment:
temp dir?

Reply:
folderName is good enough.

Reply:
No, I mean using temporary directories.

Reply:
that might hide some errors. I would prefer testing with real files.